services/entity-server/src/components/solution-map/solution-map-sidebar.js

```diff
@@ -0,0 +1,115 @@
+import style from "./solution-map-sidebar.module.css";
+import React from "react";
+import { Button, Intent } from "@blueprintjs/core";
+import { Link } from "react-router-dom";
+
+const SidebarIdeaDetails = props => {
+  const { selectedIdea } = props;
+  return (
+    <div
+      className={style.ideaDetails}
+      style={{ textAlign: selectedIdea ? "left" : "center" }}
+    >
+      {!selectedIdea && (
+        <p>Select an idea in the right window to see more information! </p>
+      )}
+      {selectedIdea && (
+        <>
+          <p>
+            <b>ID:</b> <br /> {selectedIdea.idea}
+          </p>
+          <p>
+            <b>DESCRIPTION:</b> <br /> {selectedIdea.content}
+          </p>
+          <p>
+            <b>CUSTER LABEL:</b> <br /> {selectedIdea.clusterLabel}
+          </p>
+          <Link to={"/idea/" + getNameFromUri(selectedIdea.idea)}>
```

Comment:
Link can be the same as selectedIdea.idea! Thats the benefit of having urls as ids

Reply:
this is more complicated i think, we need to have a call about this at some point i think, 

I'll leave it like this for now, if its ok.